tm1637/src/formatters.rs

```diff
@@ -0,0 +1,225 @@
+//! Format numbers into byte arrays
+//!
+//! Functions for converting numbers (i8s, i16s or i32s) into arrays of bytes
+//! that can be sent to a TM1637 display.
+//!
+//! There are versions of these functions that are meant for 4-digit displays
+//! and for 6-digit displays. The 6-digit versions take into account that the
+//! order of the bytes does not drectly correlate with the order of the physical
+//! digits.
+//!
+//! All numbers are aligned to the right.
+//!
+//! #Example
+//!
+//! ```
+//! tm.write_bytes_raw(0, i16_to_4digits(1234) );
+//! ```
+//!
+
+
+// Duplicated from mappings::DigitBits::from_digit
```

Comment:
Maybe like this?

```txt
/// Convert a digit to byte.
#[cfg_attr(
    feature = "mappings",
    doc = "Duplicated from [`DigitBits::from_digit`](crate::mappings::DigitBits::from_digit)."
)]
```

Reply:
I'm wondering if I should instead call functionality from the `mappings` module, and make the `formatters` feature depend on the `mappings` feature. Do you have any thoughts about that?

Reply:
Yes, I guess that makes sense. I thought it wasn't worth pulling in the whole ``mappings`` module just for one function. But now that you mention it, I guess it is a better idea to keep the mappings in the ``mappings`` module. Implementing other formatters like ``str_formatters`` would be cleaner this way.